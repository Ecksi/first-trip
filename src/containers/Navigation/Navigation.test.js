import React from 'react';
import { shallow } from 'enzyme';
import { Navigation } from './index';

describe('Navigation', () => {
  let navigation;
  let mockProps = {
    authUser: {}
  };

  beforeEach(() => {
    navigation = shallow(<Navigation {...mockProps} />);
  });
  
  it('should match the snapshot', () => {
    expect(navigation).toMatchSnapshot();
  });
});