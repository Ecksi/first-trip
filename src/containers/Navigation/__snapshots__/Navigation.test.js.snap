// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match the snapshot 1`] = `undefined`;
